src/components/AttachmentModal.js

```diff
@@ -157,11 +162,9 @@ class AttachmentModal extends PureComponent {
                     propagateSwipe
                 >
                     <HeaderWithCloseButton
-                        title={this.props.isUploadingAttachment
-                            ? this.props.translate('reportActionCompose.sendAttachment')
-                            : this.props.translate('common.attachment')}
+                        title={this.props.headerTitle}
```

Comment:
this should be like this.
```suggestion
                        title={this.props.headerTitle || this.props.translate('common.attachment')}
```